src/ui/forms/mainwindow.ui

```diff
@@ -311,12 +318,18 @@
   </action>
   <action name="actionFind">
    <property name="icon">
-    <iconset theme="edit-find"/>
+    <iconset theme="edit-find">
+     <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="text">
     <string>&amp;Find</string>
    </property>
   </action>
+  <action name="actionDuplicateTab">
```

Comment:
I've checked with other browsers, and neither has `Duplicate Tab` in the main menu. I think there's no need to add it here. Its place is tab's context menu, which is yet to be implemented. Just a shortcut should be fine for now.